src/main/java/com/rainbowrave/RainbowRaveMouseListener.java

```diff
@@ -0,0 +1,24 @@
+package com.rainbowrave;
```

Comment:
This whole file just delegates mouseMoved to udpateMousePositions, it should be an anonymous class in RainbowRaveMouseTrailPlugin instead of its own file.

Reply:
I updated `RainbowRaveMouseTrailPlugin` to use `MouseAdapter` as an anonymous class, can you verify that this is what you meant?

Reply:
After discussing on Discord, we realized the declaration of mouseAdapter could happen outside of startup/shutdown.